Source/WebCore/style/StyleResolver.cpp

```diff
@@ -269,11 +271,13 @@ void Resolver::addKeyframeStyle(Ref<StyleRuleKeyframes>&& rule)
     document().keyframesRuleDidChange(animationName);
 }
 
-auto Resolver::initializeStateAndStyle(const Element& element, const ResolutionContext& context) -> State
+auto Resolver::initializeStateAndStyle(const Element& element, const ResolutionContext& context, std::unique_ptr<RenderStyle> initialStyle) -> State
```

Comment:
Why isn't the `&&` needed on `std::unique_ptr<RenderStyle>` ?

Reply:
Not sure if it makes much difference but switched to that anyway.